include/fi_util.h

```diff
@@ -166,6 +166,7 @@ struct util_ep;
 typedef void (*ofi_ep_progress_func)(struct util_ep *util_ep);
 
 struct util_ep {
+	struct dlist_entry	entry;
```

Comment:
Please rename to av_entry, and move down, so that ep_fid remains the first entry in the struct.